app/views/dj_account_mailer/welcome_email.html.erb

```diff
@@ -8,23 +8,25 @@ Your account on <%= @radio.name %> has been created.
 
 <p>
 Your username is <%= @user.username %>.
-Your password is <%= @password %>. You can change your password <%== link_to "here.", edit_user_registration_url(@user) %>
+Your password is <%= @password %>.
```

Comment:
do we really need to include the password in the email?

Reply:
Definitely not if we are converting the account from listener to dj.
But
If it's a brand new account they haven't set a password yet.

Maybe we can just make them use the pw reset immediately instead?

Reply:
updated to only show PW for completely new accounts @dudethatbe